docs/index.asciidoc

```diff
@@ -20,12 +20,11 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-Implementation of a Logstash codec for the ArcSight Common Event Format (CEF)
-Based on Revision 20 of Implementing ArcSight CEF, dated from June 05, 2013
-https://community.saas.hpe.com/dcvta86296/attachments/dcvta86296/connector-documentation/1116/1/CommonEventFormatv23.pdf
+Implementation of a Logstash codec for the ArcSight Common Event Format (CEF).
+It is based on https://www.microfocus.com/documentation/arcsight/arcsight-smartconnectors/pdfdoc/common-event-format-v25/common-event-format-v25.pdf[Implementing ArcSight CEF Revision 25, September 2017].
```

Comment:
It is disappointing that the maintainers of the spec won't make it publicly available. This link works, and appears to be hosted by the maintainers of the spec, but contains a clause indicating that posession of the spec requires a license.

But: here we are. The net change is going from an http `404 Not Found` to a `200 OK`, so it is a net improvement.